### ISSUE TITLE
Nodejs version change

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#477]
As Nodejs has jumped version numbers we need to update our build code.

In progress due to these PRs:
https://github.com/turnkeylinux/common/pull/53
https://github.com/turnkeylinux-apps/etherpad/pull/4

**Comment by qrntz** [comment]
https://github.com/turnkeylinux/tracker/issues/461 is a duplicate.

**Comment by JedMeister** [comment]
I've just closed the other one and assigned this one to you! :smile: 

I note that your PRs (for Etherpad and common) are still unmerged. I was almost going to merge the Etherpad one but instead I think that it would be good to fix it properly for v14.1. 

Liraz actually did a rewrite of the Nodejs appliance (see /turnkeylinux-apps/nodejs/commits/master) so we could combine this fix with #472 (by putting the relevant parts of that code back into common). So I have just assigned that to you to ok!? FWIW he did it in the Nodejs appliance (rather than common) so he didn't inadvertently break anything; but we can take the time to do it properly now and test to ensure that it all works.
